UT4MasterServer.Web/src/types/game-server.ts

```diff
@@ -0,0 +1,57 @@
+export interface IGameServerAttributes {

```

Comment:
Since we are referencing these attributes to make decisions (like whether or not it is a hub, etc) I'd like to keep the type safety. I can remove any of the attributes that we aren't using, especially knowing now they may not be constant. Defining an interface in Typescript isn't like doing the same in C#, the JSON doesn't get serialized into the type. Typescript only exists at compile time, it doesn't actually try and create an object conforming to your interface, it is only for code completion and compilation.